.github/workflows/release.yml

```diff
@@ -52,12 +55,11 @@ jobs:
       # Needed to create the release notes
       contents: write
     steps:
-      - uses: actions/checkout@v3
-      - name: make release-notes
+      - uses: actions/checkout@v4
+      - run: make -C .ci release-notes
```

Comment:
Drop the slightly misleading `make release-notes` name: there is a separate Makefile in play.